front-end/src/MeetTheTeam.css

```diff
@@ -1,12 +1,21 @@
 h1 {
     margin-top: 0;
     padding: 0;
+    color: #D1CDC7; 
+
```

Comment:
The meet the team page looks perfect. Good work on the styling here.

Reply:
I especially like the light profile picture placeholder. That was a good idea to make.

Reply:
thanks!